pyramid_storage/gcloud.py

```diff
@@ -104,7 +126,16 @@ def _get_or_create_bucket(self, name):
         except NotFound:
             if self.auto_create_bucket:
                 bucket = self.get_connection().create_bucket(name)
-                bucket.acl.save_predefined(self.auto_create_acl)
+
+                if not self.uniform_bucket_level_access:
+                    if not self.auto_create_acl:
+                        self.logger.warning(
+                            "\"uniform_bucket_level_access\" disabled, \"auto_create_acl\" unset"
+                        )
+
+                    else:
+                        bucket.acl.save_predefined(self.auto_create_acl)
```

Comment:
You are right, the warning in this case (`auto_create_acl` falsy) is something we didn't have in the past